From dc64051357e48e9ac2985543d76ce2adaa06026b Mon Sep 17 00:00:00 2001
From: Guanhua Gao <guanhua.gao@nxp.com>
Date: Tue, 15 May 2018 17:54:13 +0800
Subject: [PATCH 475/767] Change the path of included mc.h

Change the path of included mc.h.

Signed-off-by: Guanhua Gao <guanhua.gao@nxp.com>
[Xulin: Original patch taken from NXP LSDK-18.09.]
Signed-off-by: Xulin Sun <xulin.sun@windriver.com>
---
 drivers/of/device.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/of/device.c b/drivers/of/device.c
index 0c3283262f6b..ee1f900ba6ce 100644
--- a/drivers/of/device.c
+++ b/drivers/of/device.c
@@ -15,7 +15,7 @@
 #include <asm/errno.h>
 #include "of_private.h"
 #ifdef CONFIG_FSL_MC_BUS
-#include "../staging/fsl-mc/include/mc.h"
+#include <linux/fsl/mc.h>
 #endif
 
 /**
-- 
2.17.0

